src/lazy/system_stream_item.rs

```diff
@@ -41,11 +41,11 @@ impl<'top, 'data> SystemStreamItem<'top, 'data> {
         }
     }
 
-    /// Like [`Self::version_marker`], but returns a [`crate::IonError::DecodingError`] if this item
+    /// Like [`Self::version_marker`], but returns a [`crate::IonError::Decoding`] if this item
```

Comment:
🗺️ As you'll see later, I have removed a layer of redundancy from the naming in the `IonError` type.

Now that each error variant wraps an opaque struct named `<KIND>Error`, the variant names themselves no longer include the word "Error."